### PR TITLE
Update Dockerfile and Vagrantfile for the classroom exercises

### PR DESCRIPTION
**Pull request by SudKul**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.env/
+.venv/
+.vagrant/
+.myvenv/
+__pycache__/
+.vagrant/
```
Comment (AugustasV) [inline .gitignore:6]: add a newline at the end of the file

File: exercises/README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+│   ├── deployment.yaml
+│   ├── namespace.yaml
+│   └── service.yaml
+└── python-helloworld  # Docker Walkthrough code  
+    ├── app.py
+    └── requirements.txt
+```
+
+
+**Note** that the specified `config.vm.box_version` in the Vagrantfile updates with time. You will have to change it from 242 shown in the video demo above to a newer version. When a particular version used in Vagrantfile becomes deprecated, the command prompt will show you all available newer versions.
```
Comment (kgamanji) [inline exercises/README.md:21]:
```suggestion
**Note**: the specified `config.vm.box_version` in the Vagrantfile updates with time. You will have to change it from `212` shown in the demo video above to a newer version. When a particular version used in Vagrantfile becomes deprecated, the command prompt will show you all available newer versions. 
```

File: .gitignore (hunk 1 above)
Comment (kgamanji) [inline .gitignore:6]:
```suggestion
.vagrant/

```

File: exercises/README.md [hunk 3]
```diff
@@ -1 +1,23 @@
...
+├── Vagrantfile        # Vagrant exercise
+├── go-helloworld      # Docker and Kubernetes exercise
+│   ├── README.md
+│   └── main.go
+├── manifests
+│   ├── configmap.yaml
+│   ├── deployment.yaml
+│   ├── namespace.yaml
+│   └── service.yaml
+└── python-helloworld  # Docker Walkthrough code
```
Comment (kgamanji) [inline exercises/README.md:15]:
```suggestion
└── python-helloworld  # Docker walkthrough code  
```

**Review by kgamanji — CHANGES_REQUESTED** [review]
Looks good! Just minor changes that you can easily apply as part of the review. 

Otherwise LGTM

File: project/Vagrantfile [hunk 4]
```diff
@@ -1,17 +1,27 @@
 # set up the default terminal
 ENV["TERM"]="linux"
 
+# set minimum version for Vagrant
+Vagrant.require_version ">= 2.2.10"
 Vagrant.configure("2") do |config|
+  config.vm.provision "shell",
+    inline: "sudo su - && zypper update && zypper install -y apparmor-parser"
```
Comment (kgamanji) [inline project/Vagrantfile:8]: Why is `apparmor-parser` needed?

Comment (kgamanji) [reply project/Vagrantfile:8]: Makes sense, as it's required with the higher versions: https://github.com/rancher/k3os/issues/702

**Comment by SudKul** [comment]
Thank you @kgamanji and @AugustasV for your review.

**Pull request merged**